README.Rmd

```diff
@@ -20,9 +20,9 @@ knitr::opts_chunk$set(
 [![codecov](https://codecov.io/gh/metrumresearchgroup/bbr/branch/main/graph/badge.svg)](https://codecov.io/gh/metrumresearchgroup/bbr)
 <!-- badges: end -->
 
-`bbr` is an R interface for running `bbi`. Together they provide a solution for managing projects involving modeling and simulation with a number of software solutions used in pharmaceutical sciences. Currently, only NONMEM modeling is supported, though we are in the process of Stan with plans for other modeling software as well. You can get more detailed information on `bbi` (the underlying CLI tool)  [here](https://github.com/metrumresearchgroup/bbi).
+`bbr` helps manage the entire modeling workflow from within R. Users can submit models, consume outputs and diagnostics, and iterate on models. Furthermore, workflow tools---such as simple tagging of models and model inheritance trees---make reproducibility and external review more streamlined.
```

Comment:
I know this is from the previous version, but I'm not sure how I feel about "consume outputs and diagnostics". How about "inspect" instead of "consume"? (and maybe "output" instead of "outputs"?)